test/workers/package-file/dep-type.spec.js

```diff
@@ -260,5 +260,19 @@ describe('lib/workers/package-file/dep-type', () => {
       expect(res.x).toBeUndefined();
       expect(res.packageRules).toBeUndefined();
     });
+    it('checks if currentVersion selector is valid and satisfies the condition', () => {
+      const nConfig = {
+        packageRules: [
+          {
+            matchVersion: ['^2.0.0'],
+          },
+        ],
+      };
+      const res1 = depTypeWorker.getDepConfig(nConfig, {
+        currentVersion: '1.4.6',
+      });
+      expect(res1.packageVersionMatches).toBeDefined();
```

Comment:
Does `toEqual` not do a strict equality check? I would have assumed `toEqual(false)` would check that the value was `false`, and not undefined.

Reply:
This test instead should test that the packageRule is not applied, e.g. `x` is undefined